package.json

```diff
@@ -8,7 +8,7 @@
     },
     "devDependencies": {
         "@posthog/plugin-contrib": "^0.0.3",
-        "@posthog/plugin-scaffold": "^1.0.0",
+        "@posthog/plugin-scaffold": "^1.4.0",
```

Comment:
Looks like a big bump but I think it's pretty safe as it's only types and an exception type which seems pretty low.